vignettes/laminr.Rmd

```diff
@@ -41,8 +41,9 @@ lamin connect <owner>/<name>
 ```
 
 This instance acts as the default instance for everything that follows.
+Any new records or other changes will be added here.
 
-# Start an R session
+# Connect to the default instance
```

Comment:
I find it problematic to say "connect to the default instance" in this heading, @lazappi, because that just happened 3 lines above on the command line. Don't you agree?

Reply:
This is how it reads now:

<img width="805" alt="image" src="https://github.com/user-attachments/assets/ab1debc4-feb7-49e9-87e7-e3d786bb9383">

I felt that if one calls it "Start an R session" and then people see that they create an instance object by calling `connect()` in R one avoids calling two different things (CLI connect vs R connect()) with the same name "connect to the default instance".